### PR TITLE
[Duplicate] Data synchronization and backup management in Networking

### LINKED ISSUE #6070: Data synchronization and backup management in networking
#### Proposed title of article
Data synchronization and backup management in networking

#### Proposed article introduction
Data backup is the process of creating a copy of data and storing it in a cloud based software for recovery in case the original data is lost. Data synchronization is the process of establishing consistency between two or more devices and updating changes between them. They are both used as interchangeable solutions for backing up and protecting data.
Mostly we are advised to keep our  files backed up and incase they are lost one can recover them.
Topics to be covered:

> Process of data synchronization
> Maintaining data backup
> Differences between data backup management and data synchronization
> Cloud synchronization vs cloud backup
> Data synchronization tools
> Why you should backup files

#### Key takeaways 
The learner should be able to:
> Tell the difference between data synchronization and data backup management
> Why data should be backed up
> How cloud synchronization and cloud backup works
> Process of data synchronization and how data backup is maintained

#### Article quality
Unlike other articles, this article will be unique in its own way

#### References
N/A

### Conclusion
Data backup and synchronization are used as interchangeable solutions for backing up and protecting data.

### PR DESCRIPTION
**Pull request by gladwel-kieya**
#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

#### Topic suggestion this closes

This closes #6070

### CODE REVIEW

**Comment by gladwel-kieya** [comment]
Hey @BrianaNzivu and @louisefindlay23 i've made the necessary changes

**Comment by louisefindlay23** [comment]
Plagiarism check complete. Please remember to use the correct formatting for bullet points and use line breaks for paragraphing.

**Comment by gladwel-kieya** [comment]
Hey @BrianaNzivu how's the progress

File: content/articles/data-synchronization-and-backup-management-in-networking/index.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+Types of synchronization
+server-client, which is one too many.
+Node to node/many to many here, all systems can push data.
+Sync 2-way, which is between the mobile node and a PC.
+The server-altered sync: here, the server tells the client to request data that the client pulls, e.g., email.
+Client-initiated: The client sends data, e.g., device configuration changes.
+In data synchronization, Sync-ML is a framework that provides protocols, tools, and a markup language for synchronization. WAP 2.0 makes use of it.
+
+### Patterns of Data Synchronization
+Synchronization pattern characteristics
```
Comment (BrianaNzivu) [inline content/articles/data-synchronization-and-backup-management-in-networking/index.md:27]: The first three lines of this sub-topic are unclear.

Comment (gladwel-kieya) [reply content/articles/data-synchronization-and-backup-management-in-networking/index.md:43]: I've noted the changes,,

Comment (BrianaNzivu) [reply content/articles/data-synchronization-and-backup-management-in-networking/index.md:43]: Please change them to make them clear.

Comment (gladwel-kieya) [reply content/articles/data-synchronization-and-backup-management-in-networking/index.md:43]: Hey @BrianaNzivu i've made the changes,,
Please review them

**Comment by gladwel-kieya** [comment]
Hey @BrianaNzivu how is the progress?

**Review by BrianaNzivu — APPROVED** [review]
Hey @gladwel-kieya  👋 .

Welcome to Section - EngEd.
We are happy to have you here 🚀 

Very good job on the article. It is a very interesting read 👏 . 
Just a few comments:
- I'd like to recommend Google's technical writing [course](). I believe you can improve your writing skills through this course.
- It is always advisable to submit an article that is well polished. By this, I mean having an article that does not have too many grammatical corrections and functional code. I suggest that you check out our [resources](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) page which has links to various resourceful tools for writing. 
- Please go through ALL the corrections I have made so that you can not repeat the same mistakes.

That's it! :smile: 

A final review will be done now by a content moderator.
Otherwise, congratulations on publishing your first article🍾.

In case of any question or clarification (currently or in the future), do not hesitate to reach out via [Slack](https://app.slack.com/client/T5X14SH7G/D01A6MT673R) or my personal email (briananzivu@gmail.com). 
I am **ALWAYS** here to help.

It has been a pleasure reading and reviewing this article.

Looking forward to seeing you contribute more to EngEd. 🚀

Kind Regards.

**Comment by gladwel-kieya** [comment]
hey @hectorkambow wanted to ask about the review?

**Comment by hectorkambow** [comment]
Noted: just going through the queue in order - we will get to this one asap.!

**Comment by hectorkambow** [comment]
🚀

**Pull request merged**